Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -146,7 +146,8 @@ static void updateCustomAppearance(CALayer *layer, GraphicsLayer::CustomAppearan
 #endif
 }
 
-static void applyGeometryPropertiesToLayer(CALayer *layer, const RemoteLayerTreeTransaction::LayerProperties& properties)
+enum class WithPerspectives : bool { No, Yes };
```

Comment:
I wonder if this name can be something about what we're doing to the transforms? ("FlattenPerspectiveTransforms" or something in that vein?)

Reply:
@hortont424 good call, updated!